assets/queries/k8s/limit_capabilities_from_the_container/metadata.json

```diff
@@ -1,8 +1,8 @@
 {
   "id": "limit_capabilities_of_the_container",
-  "queryName": "Limit Capabilities Of The Container",
+  "queryName": "Not Limited Capabilities For Container",
```

Comment:
Unlimited?

Reply:
Prefer the use of Not Limited in this case